source/developer/webhooks-incoming.rst

```diff
@@ -149,6 +149,15 @@ For example, to create a message with a heading, and an italicized text on the n
 
 Messages with advanced formatting can be created by including an :doc:`attachment array <message-attachments>` and :doc:`interactive message buttons <interactive-message-buttons>` in the JSON payload.
 
+Custom post type
+~~~~~~~~~~~~~~~~~~
+
+You can specify a custom post type `via client plugins <about.mattermost.com/default-plugins>`_ when sending a webhook message. To set the type, use the `type` parameter on the JSON payload.
```

Comment:
>via client plugins

You can set the post type via anything that's using the webhook/slash command/whatever. It's just that the plugins are the only thing that will use the post type right now

Maybe change this to `You can specify a custom post type when sending a webhook message, for use by <plugins>(somelinkhere). To set the type, use the `type` parameter on the JSON payload.`

I'd also leave out the "client" bit of plugins, since the server part of plugins can use the types too